src/App.tsx

```diff
@@ -193,11 +193,13 @@ class UserRelations extends React.Component<HasUserName, {}> {
   addSection = (sections, stateKey, typeKey, userNameKey, title) => {
     if (this.state[stateKey] && this.state[stateKey][typeKey]) {
       const rows = this.state[stateKey][typeKey].map(s => {
-        let userName = s.get(userNameKey);
-        let userArr =  userName.split('.');
-        let displayName = userArr[0].charAt(0).toUpperCase() + userArr[0].slice(1) + ' ' + 
-          userArr[1].charAt(0).toUpperCase() + userArr[1].slice(1);
-        return <div id={userName} key={userName}><Link to={`/m/${userName}`}>{displayName}</Link></div>;
+        let memberId = s.get(userNameKey);
+        return (
```

Comment:
Can we change `MemberTile` so `return <MemberTile memberId={memberId} />` works here?